server/fromError.js

```diff
@@ -1,14 +1,21 @@
-const { boomify } = require('boom')
-const { path, pipe, prop, unless } = require('ramda')
+const Boom = require('boom')
+const { converge, identity, path, pipe, prop, unless, when } = require('ramda')
+
+const { boomify } = Boom
 
 const formatError = ({ data, message, error: name, statusCode: status }) =>
   ({ data, message, name, status })
 
+const joiError = pipe(
+  prop('details'),
+  converge(Boom.badRequest, [ prop('message'), identity ])
```

Comment:
Let's destructure `badRequest` off of `Boom` like `boomify` is, just for consistency.